src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -39,12 +54,26 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   scheduleStartHour,
   personalizedEvent: isPersonalizedEvent = false,
 }) => {
+  const eventRef: RefObject<HTMLDivElement> = useRef(null);
+  const isLongEvent = event.duration_minutes >= minLongEventLength;
+  const isShortEvent = event.duration_minutes <= shortEventLength;
   const { userId } = useUser();
+  const [isExpanded, toggleExpanded] = useState(false);
+  const [isExpandedStatic, toggleExpandedStatic] = useState(false);
```

Comment:
We have `useShowHide` hook that can be easily used here, you can also rename the values while destructuring. Also it's default value is `false`.

Reply:
done fd981d9